scripts/camera/run-camera-service.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# This script is run from the camera pi. It will start a new camera service if there is not
+# already one running. This involves:
+# - (re)mounting the SSHFS images directory
+# - Downloading the latest docker-compose file from github
+# - Updating the relevant docker images
+# - Starting the docker camera service
+set -eu
+
+REMOTE_HOST=${HUNTSMAN_REMOTE_HOST:-${PANOPTES_CONFIG_HOST}}
+REMOTE_IMAGES_DIR=${PANUSER}@${REMOTE_HOST}:${PANDIR}/images
+LOCAL_IMAGES_DIR=${PANDIR}/images
+
+GITHUB_USER=${GITHUB_USER:-"AstroHuntsman"}
+GITHUB_BRANCH=${GITHUB_BRANCH:-"develop"}
+
+# First, check if the camera docker service is already running. If so, exit 0.
+if [ -z `docker-compose ps -q camera` ] || [ -z `docker ps -q --no-trunc | grep $(docker-compose ps -q camera)` ]; then
+  echo "No running docker camera service found. Starting a new one."
+else
+  echo "A docker camera service is already running."
+  exit 0
+fi
+
+clear
+echo "############### Huntsman Camera Service ###############"
+echo "Control computer hostname: ${REMOTE_HOST}"
+
+# Mount the SSHFS images directory
+echo "Mounting remote images directory ${REMOTE_IMAGES_DIR} to ${LOCAL_IMAGES_DIR}"
+mkdir -p ${LOCAL_IMAGES_DIR}
+sudo umount ${LOCAL_IMAGES_DIR} || true
+sshfs -o allow_other ${REMOTE_IMAGES_DIR} ${LOCAL_IMAGES_DIR}
```

Comment:
Does this stay connected if the byobu session dies?  Could we just alter fstab to have it auto mount on boot?

Also, there are various reconnect options that might be useful to specify. I will try to look up what I have used in the past.

Reply:
Doing it on boot could be something to consider. And yes, I think we implemented those in the `huntsman-pocs` SSHFS implementation.